src/components/MuiProvider/index.js

```diff
@@ -54,12 +57,17 @@ export const muiTheme = createTheme({
     },
   },
 });
+// Create rtl cache
+const cacheRtl = createCache({
+  key: 'muirtl',
+  stylisPlugins: [prefixer, rtlPlugin],
+});
 
 // MaterialUI Theme Provider only runs on the client side, so we need to wrap it in a component that only runs on the client side
 export const MuiProvider = ({ children }) => {
   return (
     <ThemeProvider theme={responsiveFontSizes(muiTheme)}>
-      {children}
+      <CacheProvider  value={cacheRtl}>{children}</CacheProvider>
```

Comment:
![image](https://github.com/BizTech-Morocco/mosa3ada.biztech.ma/assets/44750827/853474af-1365-4fb7-aace-a111f0b5224e)

This bug occurs when using the cacheProvider